targetconfig.json

```diff
@@ -121,6 +121,60 @@
             "code-ninjas-home-office/game-building-session-tutorials": {},
             "microsoft/arcade-carnival": {}
         },
+        "builtinExtensionsLib": {
+            "controller": {
+                "preferred": true,
+                "tags": [ "Utility" ]
+            },
+            "animation": {
+                "preferred": true,
+                "tags": [ "Sprites" ]
+            },
+            "multiplayer": {
+                "preferred": true,
+                "tags": [ "Utility" ]
```

Comment:
Can we add more meaningful tags to multiplayer? It spans multiple categories: Sprites, controller, game, info.

Reply:
How about just adding "Sprites" to multiplayer for now. New tags could happen in another pass at a later time.

Reply:
Oop I already did it. @eanders-ms @jwunderl do you have a sec to look through the tags? Otherwise I can revert it.

Reply:
Outside of the existing ones (`Sprite Pack`, `Hardware`, `Visual Effects`, `Sprites`, `Utility`) we oughta be pretty careful when adding new ones -- e.g. I don't know that I would expect when clicking a tag named `game` / `info`  below the search bar -- full games / docs? Controller feels like a good addition to me though, we should probably have a quick look from Jacqueline & abhijith for approval on any 'new' tags that will show up (vs just adding to existing tags)

Reply:
How does it impact the extension screen? I wonder if it might result in tag bubble overload. At a certain threshold we'll need to change that UI.

Reply:
This is what it looks like now, but I agree with Joey that `game` and `info` should probably be removed.
![image](https://user-images.githubusercontent.com/15070078/214924544-c3aec5a3-0ad5-44ee-adc3-16bba126f0b9.png)

Reply:
@abchatra @Jaqster how do you feel about adding `controller` as a tag for the extensions?

Reply:
Seems like a good tag... how many extensions would fall under "Controller"?

Reply:
These 3 for right now, but Joey also mentioned I should add the `controller` tag to the `feather` extension too. 
![image](https://user-images.githubusercontent.com/15070078/214970296-d1875719-49fe-4c6c-b90a-aeeefbf68600.png)

Reply:
Sounds good to me.